### DISCUSSION TITLE
Lock files v3 and resolved and integrity fields

### DISCUSSION THREAD

**Discussion by bozdoz** [community/community#127610] [npm] [answered]
I've noticed in a few lockfile v3 files that resolved and integrity fields can be missing. 

Why is that? Is there a way to enforce that?

**Comment by bozdoz** [comment, marked as answer]
Actually, this was the issue I was thinking of.  Guess it's still open:

https://github.com/npm/cli/issues/4460

**Comment by grandmas-favorite** [comment]
Hey @bozdoz!

Thanks for coming back to the Community to share that resource!
